spec/views/post_index_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+RSpec.describe 'Post', type: :system do
```

Comment:
* I suggest you make Specs file for the integration tests type is `feature` for example ` describe 'the signin process', type: : feature do`